File: 백준/Bronze/3003. 킹， 퀸， 룩， 비숍， 나이트， 폰/킹， 퀸， 룩， 비숍， 나이트， 폰.py
import sys

arr = [1, 1, 2, 2, 2, 8]
ins = list(map(int, sys.stdin.readline().split()))

for i in range(len(arr)):
    if (arr[i] != ins[i]):
        res = arr[i] - ins[i]
    else:
        res = 0
    print(res, end=" ")

File: 백준/Bronze/3003. 킹， 퀸， 룩， 비숍， 나이트， 폰/킹， 퀸， 룩， 비숍， 나이트， 폰.java
import java.io.*;

public class Main {
    public static void main(String[] args) throws IOException{
        int [] arr = {1, 1, 2, 2, 2, 8};
        BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
        BufferedWriter bw = new BufferedWriter(new OutputStreamWriter(System.out));
        String str = br.readLine();
        String [] ins = str.split(" ");
        int res;

        for (int i=0; i<arr.length; i++) {
            int n = Integer.parseInt(ins[i]);
            if (n != arr[i])
                res = arr[i]-n;
            else
                res = 0;
            bw.write(res + " ");
        }
        br.close();
        bw.close();

    }
}
